### ISSUE TITLE
preview not wrapping text in some circumstances

### ISSUE THREAD

**Issue by majorminors** [joshhanley/slides#15]
Code for offending slide:
```html
      <div
        class="absolute w-full h-full flex items-center justify-center transition-transform transform duration-1000 translate-x-full"
      >
        <div class="w-full h-full flex items-center justify-center">
          <div
            class="w-full h-full grid grid-cols-2 grid-rows-6 gap-1/12 p-1/12 bg-red-800 text-orange-100 relative"
          >
            <!-- quote slide -->
            <img class="absolute h-12 ml-1/12 mt-16" src="assets/img/logo-icon-white.png" />
            <p
              class="col-span-2 row-start-2 row-span-3 pr-1/4 flex items-end font-display text-4xl"
            >
              "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
              incididunt ut labore et dolore magna aliqua."
            </p>
            <div class="row-start-5 row-span-2">
              <hr class="border-t-2 border-orange-100" />
              <p class="text-2xl md:text-xl mt-4 font-bold italic">Some person.</p>
            </div>
          </div>
        </div>
      </div>
```
Screen grab:
![sIcreen](https://user-images.githubusercontent.com/19422558/80305423-6b2a0380-87b4-11ea-9c0c-2316b8f8ddf7.jpg)

**Comment by majorminors** [comment]
Will see if this is a consequence of my layout

**Comment by majorminors** [comment]
appears to be the way p elements are treated by the previewer. doesn't acknowledge formatting applied to the right (although it does to the left). Happening on content slide also.

**Comment by joshhanley** [comment]
Fixed the wrapping issue in e796a93eaea9cdc7a7ef28d4eda11e6077143687. I had to add whitespace-normal to the preview's absolutely positioned div to get it to wrap properly.
